realm-jni/src/io_realm_internal_Util.cpp

```diff
@@ -127,6 +127,9 @@ JNIEXPORT jstring JNICALL Java_io_realm_internal_Util_nativeTestcase(
     if (dotest) {
         return NULL;
     }
-    return to_jstring(env, expect);
+    string version = "Realm version " + num_to_string<int>(io_realm_internal_Version_REALM_JAVA_MAJOR) + "."
```

Comment:
Debatting with myself if it is too verbose having the "Realm version". Wouldn't just the version number be enough as it is for people copy / pasting errors to us?